tests/affine-cloud-copilot/e2e/utils/editor-utils.ts

```
@@ -282,6 +285,19 @@ export class EditorUtils {
     await page.waitForTimeout(100);
   }
 
+  public static async createDoc(page: Page, title: string, docContent: string) {
+    await clickNewPageButton(page);
+    await page.keyboard.insertText(title);
+    await this.focusToEditor(page);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)